src/EnergyPlus/DataSurfaceLists.cc

```diff
@@ -178,7 +179,6 @@ void GetSurfaceListsInputs(EnergyPlusData &state)
                     ErrorsFound = true;
                 } else { // Make sure that all of the surfaces are located in the same zone
                     state.dataSurface->SurfIsRadSurfOrVentSlabOrPool(SurfList(Item).SurfPtr(SurfNum)) = true;
-                    int ZoneForSurface; // Zone number that a particular surface is attached to
```

Comment:
CppCheck did not identify this change as a possible scope reduction so I reduced the scope too far. It's obvious now, not so much then. I'll try to be more careful but this is an easy mistake to make. I wonder what ZoneForSurface was the 2nd time through? This should have thrown a warning error unless this error was already present, which is hard to believe. It's a little scary that CI missed this. showSameZoneWarning is set correctly for each SurfaceList before the loop of surfaces so I don't see how the previous change passed.